.clang-format

```diff
@@ -2,7 +2,7 @@
 Language:        Cpp
 # BasedOnStyle:  LLVM
 AccessModifierOffset: -2
-AlignAfterOpenBracket: Align
+AlignAfterOpenBracket: true
```

Comment:
Hm. I was getting all kinds with YAML errors with this. No idea why I can't reproduce it right now. Removing